.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: Autolens Workspace Test Integration Test
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        #python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout PyAutoFit
+      uses: actions/checkout@v2
+      with:
+        repository: rhayes777/PyAutoFit
+        path: PyAutoFit
+    - name: Checkout PyAutoArray
+      uses: actions/checkout@v2
+      with:
+        repository: Jammy2211/PyAutoArray
+        path: PyAutoArray
+    - name: Checkout PyAutoGalaxy
+      uses: actions/checkout@v2
+      with:
+        repository: Jammy2211/PyAutoGalaxy
+        path: PyAutoGalaxy
+    - name: Checkout PyAutoLens
+      uses: actions/checkout@v2
+      with:
+        repository: Jammy2211/PyAutoLens
+        path: PyAutoLens
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - uses: actions/cache@v2
+      id: cache-pip
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Install dependencies
+      if: steps.cache-pip.outputs.cache-hit != 'true'
+      run: |
+        pip install Cython
+        pip install numba==0.52.0
+        pip install numpy==1.20.0
```

Comment:
numpy must already be in the requirements?

Reply:
Yes, sorry I had that in to test a fix to the numpy issue. Removed now.